packages/connected-private-route/src/index.tsx

```diff
@@ -34,10 +34,12 @@ const PrivateRoute = (props: PrivateRouteProps) => {
     authenticated,
     disableLoginProtection,
     redirectPath,
+    location,
     ...theOtherProps
   } = props;
 
-  const currentPath = props.path;
+  const currentPath = `${(location && location.pathname) || ''}${(location && location.search) ||
```

Comment:
Please add a comment describing what is being done here.

Reply:
done